manifests/init.pp

```diff
@@ -21,7 +128,7 @@
   $config_dir_group    = 'root',
   $config_dir_mode     = '0755',
   $config_ensure       = 'file',
-  $config_path         = '/etc/eyaml/config.yaml',
```

Comment:
config_path should stay the same because of the params accessing previous params as variables that I mentioned.